code/_core/obj/item/weapon/ranged/stamina_use/_stamina_use.dm

```diff
@@ -1,77 +1,77 @@
-/obj/item/weapon/ranged/stamina_use //THERE IS NO GOD HERE, ONLY CODE, HOW THE FUCK WILL THIS WORK

```

Comment:
Same issue happening in your other PRs is happening here. See https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings on how to fix this.